gym/spaces/space.py

```diff
@@ -10,18 +11,27 @@
 
 
 class Space(Generic[T_cov]):
-    """Defines the observation and action spaces, so you can write generic
-    code that applies to any Env. For example, you can choose a random
-    action.
-
-    WARNING - Custom observation & action spaces can inherit from the `Space`
-    class. However, most use-cases should be covered by the existing space
-    classes (e.g. `Box`, `Discrete`, etc...), and container classes (`Tuple` &
-    `Dict`). Note that parametrized probability distributions (through the
-    `sample()` method), and batching functions (in `gym.vector.VectorEnv`), are
-    only well-defined for instances of spaces provided in gym by default.
-    Moreover, some implementations of Reinforcement Learning algorithms might
-    not handle custom spaces properly. Use custom spaces with care.
+    """Any space inherits from this class.
```

Comment:
Personally, I think the original first line was a more helpful description of what the space class does

Reply:
Hmm I agree that it is much more concrete. I had some concerns I felt like the first sentence should point out that `Space` is essentially just an abstract superclass. I replaced it with 

> Superclass that is used to define observation and action spaces.